.github/workflows/test.yml

```diff
@@ -1,19 +1,21 @@
 name: Test / Lint
 on:
   push:
+  pull_request:
 jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-node@v2
-      with:
-        node-version: 14
-    - run: npm ci
-    - run: npm run lint
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - run: npm ci
+      - run: npm run lint
   test:
     runs-on: ${{ matrix.config.os }}
     strategy:
+      fail-fast: false
```

Comment:
I say might as well let other tests finish even if there is an error. Cover more systems faster.